docroot/themes/humsci/su_humsci_theme/scss/decanter-overrides/_decanter-overrides.scss

```diff
@@ -30,3 +30,36 @@
 h1:first-child {
   @include margin(0 null null);
 }
+
+// Embedded media in WYSIWYG.
+.embedded-entity {
```

Comment:
This is drupal specific and wont be put into decanter. we should move this to another partial